taskcluster/mozillavpn_taskgraph/transforms/scriptworker.py

```diff
@@ -39,3 +39,13 @@ def build_upstream_artifacts(config, tasks):
 
         task.setdefault("worker", {}).update(worker_definition)
         yield task
+
+@upstream_artifacts.add
+def add_authenticode_comment(config, tasks):
+    for task in tasks: 
+        artifacts = task["worker"]["upstream-artifacts"]
+        for artifact_dict in artifacts:
+            authenticode_comment = artifact_dict.get("authenticode_comment")
+            if not authenticode_comment and any(path.endswith(".msi") for path in artifact_dict["paths"]):
+                artifact_dict["authenticode_comment"]="Mozilla VPN Client installer"
```

Comment:
Per discussion on https://github.com/mozilla-mobile/mozilla-vpn-client/issues/8550#issuecomment-1854378030 / JIRA , it looks like there's a request to remove the word "Client" here.